modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/snapshot/ProposalTrustSelect.scala

```diff
@@ -19,19 +18,18 @@ import eu.timepit.refined.numeric.Positive
 object ProposalTrustSelect {
 
   def make[F[_]: MonadThrow](
-    getTrusts: F[(Option[Map[PeerId, Double]], TrustMap)],
+    getTrusts: F[(TrustLabels, TrustScores)],
```

Comment:
Can you pass `trustStorage` here? It does not make sense to pass all methods separately if you in fact use `trustStorage` in here and you wanted to make a single implementation only and iterate it over creating new strategies. 🤔

Reply:
We cannot pass the `TrustStorage`. `TrustStorage` is only available on the GL0. We want to use trust information other nodes (i.e, `currency`) as well. So, on `GL0`, `getTrusts` is created by wrapping calls to functions of `TrustStorage`. Where `TrustStorage` is unavailable, we populate `getTrust` from the `TrustRoutes`.